src/c++/uds/src/uds/permassert.h

```diff
@@ -6,6 +6,7 @@
 #ifndef PERMASSERT_H
 #define PERMASSERT_H
 
+#include <linux/build_bug.h>
```

Comment:
Mike's original line doesn't add this #include line. is it really necessary? What happens to the kernel build if you remove it?